troposphere/static/js/components/modals/image_version/ImageVersionEditModal.react.js

```diff
@@ -159,9 +167,23 @@ export default React.createClass({
     // Render
     // ------
     //
+    validateName: function(image, name) {
+        let orig_version = this.props.version,
+        versions = image.get('versions');
```

Comment:
I think Line 172 & 174 should be indented to show they're part of the `let` statement on 171.